Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/smart_gun.yml

```diff
@@ -34,3 +34,11 @@
         Piercing: 30
   - type: CMArmorPiercing
     amount: 0
+  - type: RMCProjectileDamageFalloff
+    thresholds:
+    - range: 12
+      falloff: 9999
+      ignoreModifiers: true
+    - range: 7
+      falloff: 4
```

Comment:
Are these the updated values from 13's most recent PR?

Reply:
Yep. The max range was 12 before the PR, and it had the normal bullet stats otherwise. It was only the effective range that was changed to 7 and falloff from it to 4.

Reply:
I'll merge this probably with either the IO update or motion detectors if we get them before then, I want to add job slot scaling in the same update that removes autobalancing and I don't want that to screw with the numbers